### DISCUSSION TITLE
Copying project files on build

### DISCUSSION THREAD

**Discussion by kajoel** [James-Yu/LaTeX-Workshop#3823] [Q&A]
Is it possible to temporarily copy the project files when building to enable editing while compiling without interfering with the build process?

If it matters, I'm using a custom "Out Dir" but other than that my set-up is pretty standard (using latexmk to build).

**Comment by jlelong** [comment]
No it is not.

**Reply by kajoel** [reply]
Is this something you're planning to implement? If not, is there a good reason not to implement it?

**Reply by jlelong** [reply]
No, this is out of the scope of the extension. 
Actually, I cannot really see what you are trying to do. As long as you do not save the file, you can keep on editing it without interfering with the building process.

**Reply by kajoel** [reply]
Okay, thanks for the explanation. Changing my autosave setting might be a good enough workaround.
